README.md

```diff
@@ -9,17 +9,40 @@ This repository contains a collection of engines that power the core stack for
 Client](https://www.prisma.io/client) and [Prisma
 Migrate](https://www.prisma.io/migrate).
 
-The engines and their respective binary crates are:
+If you're looking for how to install Prisma or any of the engines, the [Getting
+Started](https://www.prisma.io/docs/getting-started) guide might be useful.
 
-- Query engine: `query-engine`
-- Migration engine: `migration-engine-cli`
-- Introspection engine: `introspection-engine`
-- Prisma Format: `prisma-fmt`
+This document describes some of the internals of the engines, and how to build
+and test them.
+
+## What's in this repository
```

Comment:
This is great. 

I mined this exhaustive graph of dependencies (redundancy removed) And want to at least document the main responsibility for each of them, but it would be challenging to keep up to date. 

![dedup_deps](https://user-images.githubusercontent.com/210307/196735321-cfd43d1a-b82f-4929-9d15-86bfb3b88e4c.png)

I think this high level overview of the main engines, prisma format and psl is good enough for now. Thank you so much for adding them!